traiNNer/models/omnisr_model.py

```diff
@@ -0,0 +1,283 @@
+import numpy as np
+import random
+import torch
+from collections import OrderedDict
+from torch.nn import functional as F
+
+from ..data.degradations import random_add_gaussian_noise_pt, random_add_poisson_noise_pt
+from ..data.transforms import paired_random_crop
+from ..losses.loss_util import get_refined_artifact_map
+from ..models.srgan_model import SRGANModel
+from ..utils import DiffJPEG, USMSharp
+from ..utils.img_process_util import filter2D
+from ..utils.registry import MODEL_REGISTRY
+
+
+@MODEL_REGISTRY.register(suffix='traiNNer')
+class OmniSRModel(SRGANModel):
```

Comment:
Is this necessary? Does it do anything differently from the regular SRGANModel class?

Reply:
no, just copy just copy hhh